theme/layout.html

```diff
@@ -7,11 +7,11 @@
 {%- if url_root == '#' %}{% set url_root = '' %}{% endif %}
 
 {%- macro css() %}
-  <link href="{{ pathto('_static/css/reset-min.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/base.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/code.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/colors.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/pygments.css', 1) }}" rel="stylesheet" type="text/css" />
+  <link href="/_static/css/reset-min.css" rel="stylesheet" type="text/css" />
```

Comment:
i see value in this change, but it also make it not usable for local development.
(`ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer.github.io λ open dist/index.html` - and no assets are found)

any suggestion for this?

Reply:
Done, this PR deals only with title now